xcm/xcm-executor/src/lib.rs

```diff
@@ -498,9 +497,7 @@ impl<Config: config::Config> XcmExecutor<Config> {
 	}
 
 	fn reanchored(mut assets: Assets, dest: &MultiLocation) -> Result<MultiAssets, XcmError> {
```

Comment:
Since `reanchor` is infallible now, there's no need to return a `Result` anymore.

Reply:
Yeah, we should replace assets which cannot be reanchored back into Holding to avoid crazy behaviour.

Reply:
Done.